src/components/molecules/bar-chart.tsx

```diff
@@ -85,12 +86,25 @@ export const TrackerBarChart: FC<TrackerBarChartProps> = ({
     return null;
   }
 
-  const last = chartData[chartData.length - 1];
-  const labelString = `${last} ${yesterday}`;
+  const isAxisLabelHidden = (index: number, showIfLessDataThan: number = 0) =>
+    chartData.length > showIfLessDataThan &&
+    index % intervalsCount &&
+    index !== axisData.length - 1;
+
+  const dataSummary = axisData.reduce((summaryText, date, index) => {
+    if (isAxisLabelHidden(index)) {
+      return summaryText;
+    }
+    const stringStart = summaryText ? summaryText + ', ' : '';
+    return `${stringStart}${format(date, 'MMMM')} ${format(
+      date,
+      'do'
+    )}: ${roundNumber(chartData[index])}${ySuffix}`;
+  }, '');
```

Comment:
Would this not be better to be `', '` to remove the `stringStart` ternary?

Reply:
Unless I've misunderstood what you're suggesting @ShaunBaker wouldn't that that give us `, ` at the start of the string? It adds `', '` if it's not the (falsey, empty) first iteration. 

But it could be cleaner written than it is, I'll tidy it up.

Reply:
Doh! thoughts in reverse lol